web/content/js/nav.js

```diff
@@ -19,37 +19,25 @@ let navVersions = [
 
 let navItems = [
 {
-	title: "Foreman documentation", items: [
+	title: "Foreman on EL", items: [
 	{
-		title: "Planning for Foreman",
-		href: "/Planning_Guide/index-foreman-el.html"
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-el.html"
 	},{
-		title: "Quickstart Guide on RHEL/CentOS",
+		title: "Quickstart Guide",
 		href: "/Quickstart_Guide/index-foreman-el.html"
 	},{
-		title: "Quickstart Guide on Debian/Ubuntu",
-		href: "/Quickstart_Guide/index-foreman-deb.html"
-	},{
-		title: "Installing on RHEL/CentOS",
+		title: "Installation Guide",
 		href: "/Installing_Server_on_Red_Hat/index-foreman-el.html"
 	},{
-		title: "Installing Smart Proxy on RHEL/CentOS",
+		title: "Installing Smart Proxy",
 		href: "/Installing_Proxy_on_Red_Hat/index-foreman-el.html"
-	},{
-		title: "Installing on Debian/Ubuntu",
-		href: "/Installing_Server_on_Debian/index-foreman-deb.html"
-	},{
-		title: "Installing Smart Proxy on Debian/Ubuntu",
-		href: "/Installing_Proxy_on_Debian/index-foreman-deb.html"
 	},{
 		title: "Upgrading and Updating",
 		href: "/Upgrading_and_Updating/index-foreman-el.html"
```

Comment:
I don't think this guide is ready for public consumption. It is very Katello centric. It's really crucial once we go live, but  at this point it's not usable.

Reply:
After https://github.com/theforeman/foreman-documentation/pull/974 was merged, I looked at more chapters that can be skipped: https://github.com/theforeman/foreman-documentation/pull/976 takes a first stab at it. However, I haven't gotten to the meat of the upgrade: the actual upgrade commands. I think they may be broken for all of upstream, but at least for Debian you can't use foreman-maintain.

Reply:
I will only keep Katello version for now then.